### PR TITLE
remove dashboard link

### PR DESCRIPTION
**Pull request by mahendra-egov**
https://digit-discuss.atlassian.net/browse/FSM-247

### CODE REVIEW

File: packages/modules/fsm/src/components/inbox/FSMLink.js [hunk 1]
```diff
@@ -6,18 +6,17 @@ import { useTranslation } from "react-i18next";
...
       text: t("ES_TITLE_NEW_DESULDGING_APPLICATION"),
       link: "/digit-ui/employee/fsm/new-application",
-      // accessTo: ["CSR"]
-      notAccessTo: [COLLECTOR],
+      accessTo: ["FSM_CREATOR_EMP"],
     },
     // { text: t("ES_TITLE_REPORTS"), link: "/employee" },
-    { text: t("ES_TITLE_DASHBOARD"), link: DSO ? "/digit-ui/employee/fsm/dso-dashboard" : "/employee", hyperlink: !DSO },
+    // { text: t("ES_TITLE_DASHBOARD"), link: DSO ? "/digit-ui/employee/fsm/dso-dashboard" : "/employee", hyperlink: !DSO },
   ];
```
Comment (harit-egov) [inline packages/modules/fsm/src/components/inbox/FSMLink.js:20]: dashboard link seems commented

File: packages/modules/fsm/src/components/inbox/FSMLink.js [hunk 2]
```diff
@@ -68,8 +67,13 @@ const FSMLink = ({ isMobile, data }) => {
       <div className="complaint-links-container">
         {GetLogo()}
         <div className="body">
-          {links.map(({ link, text, hyperlink = false, notAccessTo = [] }, index) => {
-            if (!notAccessTo?.includes(COLLECTOR)) {
+          {links.map(({ link, text, hyperlink = false, accessTo = [] }, index) => {
```
Comment (harit-egov) [inline packages/modules/fsm/src/components/inbox/FSMLink.js:70]: no role based filtering applied as of now

Comment (mahendra-egov) [reply packages/modules/fsm/src/components/inbox/FSMLink.js:70]: pls check below code

File: packages/modules/fsm/src/components/inbox/FSMLink.js (hunk 1 above)
Comment (mahendra-egov) [reply packages/modules/fsm/src/components/inbox/FSMLink.js:20]: this is what the task was

**Review by saurabh-egov — APPROVED** [review]

**Pull request merged**